crates/ruff_db/src/diagnostic/mod.rs

```diff
@@ -1016,6 +1032,17 @@ pub enum DiagnosticId {
 
     /// Use of a deprecated setting.
     DeprecatedSetting,
+
+    /// The code needs to be formatted.
+    Unformatted,
+
+    /// Range formatting isn't supported for notebooks.
+    RangeFormatNotebook,
```

Comment:
I don't think it's worth having a diagnostic ID for every CLI option that can possibly be invalid. Maybe `invalid-cli-option`?